src/app/server/Mdns.cpp

```diff
@@ -251,13 +251,15 @@ CHIP_ERROR MdnsServer::AdvertiseOperational()
         {
             uint8_t mac[8];
 
-            const auto advertiseParameters = chip::Mdns::OperationalAdvertisingParameters()
-                                                 .SetPeerId(fabricInfo.GetPeerId())
-                                                 .SetMac(FillMAC(mac))
-                                                 .SetPort(GetSecuredPort())
-                                                 .SetMRPRetryIntervals(CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL,
-                                                                       CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL)
-                                                 .EnableIpV4(true);
+            const auto advertiseParameters =
+                chip::Mdns::OperationalAdvertisingParameters()
+                    .SetPeerId(fabricInfo.GetPeerId())
+                    .SetMac(FillMAC(mac))
+                    .SetPort(GetSecuredPort())
+                    // TODO: This uses active twice, but we don't have an idle default. Is this as-intended?
```

Comment:
We do have `CHIP_CONFIG_MRP_DEFAULT_INITIAL_RETRY_INTERVAL`. Could you use it instead?

Reply:
Do you mean to use CHIP_CONFIG_MRP_DEFAULT_INITIAL_RETRY_INTERVAL for both? Or just for the idle value?

Reply:
@turon - do you know how these should be set? What should the idle default be?

Reply:
Currently `MRP_RETRY_INTERVAL_IDLE` is the new name for `INITIAL`.  The reason is that long term we'd like the logic to track whether a peer is Idle or Active outside of any given exchange and not just blindly reset to the INITIAL timeout at the start of any exchange.

Also, the spec gives default values for each optimized for "powered" nodes:

Parameter | Description | Default
MRP_RETRY_INTERVAL_IDLE | Initial base interval for retransmission. | 300 milliseconds
MRP_RETRY_INTERVAL_ACTIVE | Base interval for retransmission to an active peer. | 300 milliseconds

If the node is using the default parameters, CRI and CRA can be omitted from mdns accordingly.

Reply:
Thanks - ok, so I'll use initial for idle then. For now, I'm going to leave this in the mdns even for default - I suppose we can add more sophisticated logic in the caller later to only add if the current value isn't the default.

Reply:
Updated in latest commit.